tests/formats/pdb.cpp

```diff
@@ -228,10 +273,11 @@ TEST_CASE("Read files in PDB format") {
 
     SECTION("Read atomic insertion codes") {
         auto frame = Trajectory("data/pdb/insertion-code.pdb").read();
+        auto& topo = frame.topology();
 
-        CHECK(frame[0].get<Property::STRING>("insertion_code").value() == "a");
-        CHECK(frame[1].get<Property::STRING>("insertion_code").value() == "c");
-        CHECK(frame[2].get<Property::STRING>("insertion_code").value() == "x");
+        CHECK(topo.residue_for_atom(0)->get("insertion_code")->as_string() == "a");
```

Comment:
Looks like this is now a residue property instead of an atomic property, this should be updated in the docs!

Reply:
It was already a residue property! This test was wrong!